TESTING.md

```diff
@@ -318,6 +319,56 @@ If JS expression, following `@ts-test-if:` evaluates to truthy value, `@ts-expec
 inserted in its place during the generation. All parameters from the test matrix can be used within that
 expression.
 
+## Memory tests
+
+This suite tests client for memory leaks. It works by repeatedly running test code in a loop
+and monitoring V8 heap usage after every iteration. If it then detects that memory usage growth
+with a rat, higher then threshold it will report memory leak and file the test.
```

Comment:
```suggestion
with a rate, higher then threshold it will report memory leak and file the test.
```
Nit

Reply:
What does this means by the way `and file the test`

Reply:
It means I can't type, it should've been `fail` :D